DiagManager/CustomDiagnostics/SQL Server Perf Stats/SQL Server Perf Stats Snapshot.sql

```diff
@@ -225,64 +225,119 @@ begin
 
 	--new stats DMV
 	set nocount on
-	declare @dbname sysname, @dbid int	
+	declare @dbname sysname, @dbid int,  @sql_major_version INT, @sql_major_build INT
+
+	SELECT @sql_major_version = (LEFT(CAST(SERVERPROPERTY(N'ProductVersion') AS NVARCHAR(128)), (CHARINDEX(N'.', CAST(SERVERPROPERTY(N'ProductVersion') AS NVARCHAR(128)))-1))), 
```

Comment:
Could we replace this line with something like this:

SELECT @sql_major_version = (CAST(PARSENAME(CAST(SERVERPROPERTY('ProductVersion') AS varchar(20)), 4) AS INT)) 

It is going to be consistent with the second line, it casts the value directly into INT as opposed to relying on an implicit cast and is overall more efficient.